tests/api/v1/subjects/delete.js

```diff
@@ -37,8 +37,8 @@ describe('tests/api/v1/subjects/delete.js >', () => {
     after(tu.forceDeleteUser);
 
     describe('Childless Subjects >', () => {
-      const n = { name: `${tu.namePrefix}NorthAmerica` };
-      let i = 0;
+      const subject = { name: `${tu.namePrefix}NorthAmerica` };
+      let isubjectId = 0;
```

Comment:
If you prefer more verbose variable names, please make sure they are clear. Why "isubjectId" and not just "subjectId" or "id"?

Reply:
yeap, my mistake, should be subject Id.